docs/SETUP-GUIDE.md

```diff
@@ -0,0 +1,168 @@
+# 🚀 Apolusia Setup Guide
+
+This guide provides instructions for setting up the Apollusia project using Docker Compose or on a local Linux environment.
+
+## Table of Contents
+
+- [Docker Compose Setup](#-docker-compose-setup)
+- [Local Linux Environment Setup](#-local-linux-environment-setup)
```

Comment:
I think there might be a more optimal setup with docker-only so it's less work to setup for users. I would suggest that we delete this section and add it later, when I have a full setup-guide for it. Currently the guide seems to introduce security issues with the mentioned versions (e.g. Keycloak <- we are currently on 25+), so proper setup is necessary. The `.env`-file can also be included via `docker-compose` e.g.

```docker
  backend:
    image: ghcr.io/morphclue/apollusia-backend:latest
    ports:
      - 3000
    depends_on:
      - mongo
    env_file:
      - backend.env
```
I've opened up #198 for this.